src/routes/landing/components/CityEntry.js

```diff
@@ -60,18 +74,28 @@ class CityEntry extends React.PureComponent<PropType> {
     return (
       <CityListItem onPress={this.navigateToDashboard}
                     underlayColor={theme.colors.backgroundAccentColor}>
-        <View>
+        <Labels>
           <Label theme={theme} searchWords={[filterText]} textToHighlight={city.name} sanitize={normalizeSearchString}
-                 highlightStyle={{ backgroundColor: theme.colors.themeColor }} />
+                 highlightStyle={{ fontWeight: 'bold' }} />
           {aliases.length > 0 && <Aliases>
-            {aliases.map(
-              (alias, index) => <AliasLabel key={alias} theme={theme} searchWords={[filterText]}
-                                            textToHighlight={index === aliases.length - 1 ? alias : `${alias}, `}
-                                            sanitize={normalizeSearchString}
-                                            highlightStyle={{ backgroundColor: theme.colors.themeColor }} />
+            {aliases.slice(0, MAX_NUMBER_OF_ALIASES_SHOWN).map(
+              (alias, index) => <>
+                <AliasLabel key={alias} theme={theme} searchWords={[filterText]}
+                            textToHighlight={alias}
+                            sanitize={normalizeSearchString}
+                            highlightStyle={{ fontWeight: 'bold' }} />
+                {index !== aliases.slice(0, MAX_NUMBER_OF_ALIASES_SHOWN).length - 1 && <>
```

Comment:
I think it would be good style to avoid the duplicate `aliases.slice(0, MAX_NUMBER_OF_ALIASES_SHOWN)`. This could be achieved by overwriting aliases and having a boolean:
```
slice_needed = aliases.length > MAX_NUMBER_OF_ALIASES_SHOWN
aliases = aliases.slice(0, MAX_NUMBER_OF_ALIASES_SHOWN)
```